api/src/middleware/achievements.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * This module contains middleware functions for the achievements route (../routes/achievements.js).
+ */
+
+const { body, validationResult } = require('express-validator');
+const { fillErrorObject } = require('./error');
+
+/**
+* Handles the validation when creating (POST) a new achievement in the database.
+* There are validators for: title, description.
+* @returns 400 if validation for any attribute fails, otherwise move to next function in the chain
+*/
+const createAchievementValidation = [
```

Comment:
Did we want to run validation for the year as well, seeing as that's a `Number`? We could just check to see that the year is valid format, we probably don't have to go into checking whether the year is before the current day/any complex validation